File: Pokemondle FrontEnd/Pokemondle/src/app/feedback/feedback.ts
export enum Answer{
    Greater='Greater',
    Equal='Equal',
    Lower='Lower'
}

 export class Feedback {
    constructor(
        public checkName: boolean,
        public checkGeneration: Answer,
        public checkType1: boolean,
        public  checkType2: boolean,
        public checkHeight: Answer,
        public checkWeight: Answer,
        resultName: string
    ){}
}

File: Pokemondle Backend/src/main/java/com/example/pokemondle/model/Feedback.java
package com.example.pokemondle.model;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class Feedback {
    boolean checkName;
    Answer checkGeneration;
    boolean checkType1;
    boolean checkType2;
    Answer checkHeight;
    Answer checkWeight;
}
